README.md

```diff
@@ -71,11 +77,11 @@ curl -L https://install.meilisearch.com | sh
 ./meilisearch --master-key=masterKey
 ```
 
-NB: you can also download Meilisearch from **Homebrew** or **APT** or even run it using **Docker**.
+You can also install Meilisearch using [Homebrew](https://brew.sh), [APT](https://ubuntu.com/server/docs/package-management), or [Docker](https://www.docker.com/). Consult the official documentation for more information on [downloading and running Meilisearch](https://docs.meilisearch.com/reference/features/installation.html#download-and-launch).
```

Comment:
I think we don't need to list all the ways to download meilisearch. Just the link to the documentation is nice.

What do you think?

Reply:
Always up for cutting down text.

What do you think about removing the example above describing how to install Meilisearch locally while we're at it? I'm a bit afraid that showing an example of local installs with `curl` might make users think this is our recommendation as the best way of running Meilisearch

Reply:
I agree let's cut the text and the `curl` example a link to the doc will be good enough

Reply:
Just the link to the docs is enough!